partials/Dockerfile.rstudio.partial

```diff
@@ -0,0 +1,10 @@
+# Install RStudio
+ENV S6_VERSION={{S6_VERSION}}
+ENV RSTUDIO_VERSION={{RSTUDIO_VERSION}}
+RUN /rocker_scripts/install_rstudio.sh
```

Comment:
um, does this work? there's no `COPY` in this Dockerfile...

Reply:
See above.